debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerHelper.java

```diff
@@ -49,34 +48,37 @@
      * @param archiveLogOnlyMode true to mine only archive lgos, false to mine all available logs
      * @param archiveDestinationName configured archive log destination name to use, may be {@code null}
      * @param maxRetries the number of retry attempts before giving up and throwing an exception about log state
+     * @param initialDelayMs the initial delay
+     * @param maxDelayMs the maximum delay
      * @throws SQLException if anything unexpected happens
      * @return current redo log sequences
      */
     // todo: check RAC resiliency
     public static List<BigInteger> setLogFilesForMining(OracleConnection connection, Scn lastProcessedScn, Duration archiveLogRetention,
-                                                        boolean archiveLogOnlyMode, String archiveDestinationName, int maxRetries)
-            throws SQLException, InterruptedException {
+                                                        boolean archiveLogOnlyMode, String archiveDestinationName, int maxRetries,
+                                                        Duration initialDelay, Duration maxDelay)
+            throws SQLException {
         List<BigInteger> ret = new LinkedList<>();
         removeLogFilesFromMining(connection);
 
         // Restrict max attempts to 0 or greater values (sanity-check)
         // the code will do at least 1 attempt and up to maxAttempts extra polls based on configuration
         final int maxAttempts = Math.max(maxRetries, 0);
+        final DelayStrategy retryStrategy = DelayStrategy.exponential(initialDelay.toMillis(), maxDelay.toMillis());
```

Comment:
We should add an overload for `exponential()` to accept `Duration` too (or better yet, just change it). Can happen separately though.

Reply:
Well, it is a separate task. *All* factory methods need to be updated.

Reply:
Yep, it's tracked via a follow-up issue now.